src/amo/pages/SearchTools/index.js

```diff
@@ -29,41 +19,10 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
-  _config: typeof config,
-  clientApp: string,
-  dispatch: DispatchFunc,
   i18n: I18nType,
-  lang: string,
 |};
 
 export class SearchToolsBase extends React.Component<InternalProps> {
-  static defaultProps = {
-    _config: config,
-  };
-
-  constructor(props: InternalProps) {
-    super(props);
-
-    // See: https://github.com/mozilla/addons-frontend/issues/8679
-    if (props._config.get('enableFeatureRemoveSearchTools')) {
```

Comment:
We need to keep the logic inside the `if` block now. This is a redirect to support old URLs.